oauth-proxy/values.yaml

```diff
@@ -0,0 +1,102 @@
+## Default values for oauth-proxy service
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## Count of instances
+## THIS SERVICE IS NOT READY TO BE REPLICABLE, SO PLEASE USE 1 INSTANCE HERE
+##
+replicaCount: 1
+
+## Count of revisions saved in release history
+##
+revisionHistoryLimit: 5
+
+## Time of accepting requests to the service ( seconds )
+##
+minReadySeconds: 10
+
+## Value which provide max unavailable state ( percents )
+##
+maxUnavailable: 25%
+
+## Max count of surge replicas
+##
+maxSurge: 1
+
+image:
+  ## Docker Registry/Hub image name and tag
+  ##
+  registry: registry.k8s.community
+  name: oauth-proxy
+  tag: 0.0.1
+
+  ## Docker Registry/Hub auth secret name, always use `registry-pull-secret` if registry inside if k8s
+  ##
+  pullSecretName: registry-pull-secret
+
+  ## Docker Registry/Hub image pull policy
+  ##
+  pullPolicy: IfNotPresent
+
+ingress:
+  ## If true, Ingress will be created
+  ##
+  enabled: true
+
+  ## Ingress annotations
+  ##
+  # annotations:
+  #   kubernetes.io/ingress.class: nginx
+  #   kubernetes.io/tls-acme: 'true'
+
+  ## Ingress hostnames
+  ## Must be provided if Ingress is enabled
+  ##
+  hosts:
+  - services.k8s.community
+
+  ## Ingress path feature allow access to service through path 
+  ## example: https://hostname/namespace/servicename
+  ##
+  usePath: true
+
+  ## Ingress TLS configuration
+  ## Secrets must be manually created in the namespace
+  ##
+  tls:
+  - secretName: tls-secret
+    hosts:
+    - services.k8s.community
+
+service:
+  ## App container name
+  ##
+  name: oauth-proxy
+  
+  ## Service Type
+  ## For minikube, set this to NodePort, elsewhere use ClusterIP
+  ##
+  type: ClusterIP
+
+  ## App service port
+  ##
+  externalPort: 80
+
+  ## Pod exposed port
+  ##
+  internalPort: 8080
+
+  ## Health cheks port
+  ##
+  healthPort: 8080
```

Comment:
`healthPort` separated value  is not necessary in 90% of cases, you can remove it.